app/javascript/articles/components/TagList.jsx

```diff
@@ -1,8 +1,9 @@
 import { h } from 'preact';
+import PropTypes from 'prop-types';
 import { tagPropTypes } from '../../src/components/common-prop-types';
 
-export const TagList = ({ tags = [] }) => (
-  <div className="tags">
+export const TagList = ({ tags = [], className }) => (
+  <div className={`tags${className ? ` ${className}` : ''}`}>
```

Comment:
This change is for the `<FeaturedArticle />` which takes additional CSS classes.